### PR TITLE
Fix Invalid if statement

### PR DESCRIPTION
**Pull request by mengjiao-liu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
 "if/else if" statements should not have the same condition

A chain of `if/else if `statements is evaluated from top to bottom. At most, only one branch will be executed: the first one with a condition that evaluates to `true`.

Therefore, duplicating a condition automatically leads to `dead code`. Usually, this is due to a copy/paste error. At best, it's simply dead code and at worst, it's a bug that is likely to induce further bugs as the code is maintained, and obviously it could lead to unexpected behavior.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: edge/pkg/edged/volume/csi/csi_client.go [hunk 1]
```diff
@@ -217,7 +217,7 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 		var getNodeInfoError error
 		if c.nodeV1ClientCreator != nil {
 			nodeID, maxVolumePerNode, accessibleTopology, getNodeInfoError = c.nodeGetInfoV1(ctx)
-		} else if c.nodeV1ClientCreator != nil {
+		} else {
```
Comment (gy95) [inline edge/pkg/edged/volume/csi/csi_client.go:220]: `c.nodeGetInfoV1` will assign values to `c.nodeV1ClientCreator`, so the second `if` statement may be different from the first `if`

File: edge/pkg/edged/volume/csi/csi_client.go [hunk 2]
```diff
@@ -217,7 +217,7 @@ func (c *csiDriverClient) NodeGetInfo(ctx context.Context) (
 		var getNodeInfoError error
 		if c.nodeV1ClientCreator != nil {
 			nodeID, maxVolumePerNode, accessibleTopology, getNodeInfoError = c.nodeGetInfoV1(ctx)
-		} else if c.nodeV1ClientCreator != nil {
```
Comment (fisherxu) [inline edge/pkg/edged/volume/csi/csi_client.go:220]: `} else if c.nodeV0ClientCreator != nil {` ?

**Review by fisherxu — COMMENTED** [review]
Please fix the DCO iossue.

File: edge/pkg/edged/volume/csi/csi_client.go (hunk 2 above)
Comment (mengjiao-liu) [reply edge/pkg/edged/volume/csi/csi_client.go:220]: but the struct have no the field `c.nodeV0ClientCreator`
https://github.com/kubeedge/kubeedge/blob/f7c9abcebc5cf6b3825785b9dfffec9977bec55f/edge/pkg/edged/volume/csi/csi_client.go#L109-L114

git commit 56f91bf6b3bf2e613b7ab3af2df93b692b1cf3ff remove this field `nodeV1ClientCreator`

My guess is that PR didn't delete this else if statement when we updated the code in bulk

The V0 version is no longer in use

Comment (mengjiao-liu) [reply edge/pkg/edged/volume/csi/csi_client.go:220]: Maybe it needs to be cleaned up again

Comment (fisherxu) [reply edge/pkg/edged/volume/csi/csi_client.go:220]: Looks Good.

**Pull request merged**